src/lib/pages/query/components/SchemaQueryComponent.tsx

```diff
@@ -245,7 +250,10 @@ export const SchemaQueryComponent = ({
                   <Button
                     variant="primary"
                     size="sm"
-                    onClick={handleQuery}
+                    onClick={() => {
+                      handleQuery();
+                      track(AmpEvent.USE_JSON_QUERY_AGAIN);
```

Comment:
USE_JSON_QUERY_NO_INPUT? since if we query the one with input "again" we won't track those events